.github/workflows/publish.yml

```diff
@@ -26,6 +26,8 @@ jobs:
       run: cargo publish --manifest-path Cargo.toml -p akd
       env:
         CARGO_REGISTRY_TOKEN: ${{ secrets.crates_io_token }}
+    - name: Run the script that waits for AKD to be published
+      run: ./.github/workflows/wait-for-akd-publish.sh
```

Comment:
(Just checking) you're sure the run-directory of the CI pipeline is the repo root and not something wonky like ```.github/workflows/```?

Reply:
Yes, verified that CI uses the repo root as the run-directory.